go.mod

```diff
@@ -26,7 +26,7 @@ require (
 	github.com/onsi/ginkgo/v2 v2.2.0
 	github.com/onsi/gomega v1.20.2
 	github.com/opencontainers/go-digest v1.0.0
-	github.com/opencontainers/image-spec v1.0.3-0.20220114050600-8b9d41f48198
+	github.com/opencontainers/image-spec v1.1.0-rc1
```

Comment:
just doublechecking we're ok with rc1 vendoring

Reply:
This was vendored into containers/image so I think yes. Prior to this we had an unreleased version.